### DISCUSSION TITLE
Install XDM on Ubuntu???

### DISCUSSION THREAD

**Discussion by DiabloGamer** [subhra74/xdm#716] [Q&A]
I am new to Linux. How do I install the downloaded file xdm-setup-7.2.11.tar.xz ?
Isn't there a snap for it?
Thanks for helping me!

**Comment by hamdanltf** [comment]
you can install by extracting the xdm-setup-7.2.11.tar.xz, open the extracted directory In the terminal, and run:

sudo ./install.sh

there is a README in the extracted directory
hope it's helpful
